torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -238,6 +238,7 @@ void IndexCompute::handle(Split* split) {
 
   if (outer_zero && inner_zero) {
     index_map_[in_id] = new kir::Int(0);
+    extent_map_[in_id] = new kir::Int(0);
   } else if (outer_zero) {
```

Comment:
Shouldn't we initialize `extent_map_[in_id] = new kir::Int(0)` in the two `else if` block as well?

Reply:
I think in those cases the extent map for `in_id` should inherit from the non-zero index, so I think the current code is correct.